### PR TITLE
grass.jupyter: Modify Descriptions

### PR DESCRIPTION
**Pull request by 29riyasaxena**

### CODE REVIEW

File: python/grass/jupyter/utils.py [hunk 1]
```diff
@@ -225,7 +226,14 @@ def save_vector(name, geo_json):
 
 
 def get_number_of_cores(requested, env=None):
-    """Get the number of cores to use for multiprocessing."""
+    """
+    Get the number of cores to use for multiprocessing.
```
Comment (petrasovaa) [inline python/grass/jupyter/utils.py:230]: Just a small detail, but please format it this way (see [Sphinx docstring format](https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html))
```suggestion
    """Get the number of cores to use for multiprocessing.
```

File: python/grass/jupyter/baseseriesmap.py [hunk 2]
```diff
@@ -1,3 +1,15 @@
+#
+# AUTHOR(S): Riya Saxena <29riyasaxena AT gmail>
+#
+# PURPOSE:   This module contains functions for interactive visualizations
+#            in Jupyter Notebooks.
```
Comment (petrasovaa) [inline python/grass/jupyter/baseseriesmap.py:5]: This could be more about baseseriesmap, no?

File: python/grass/jupyter/utils.py [hunk 3]
```diff
@@ -225,7 +225,13 @@ def save_vector(name, geo_json):
 
 
 def get_number_of_cores(requested, env=None):
-    """Get the number of cores to use for multiprocessing."""
+    """Get the number of cores to use for multiprocessing.
+
+    :param int requested: Desired number of cores.
+    :param dict env: Optional environment dictionary with an "NPROCS" key.
```
Comment (petrasovaa) [inline python/grass/jupyter/utils.py:231]: I know this is confusing, but this `env` doesn't contain `NPROCS`
```suggestion
    :param dict env: Optional process environment.
```

File: python/grass/jupyter/baseseriesmap.py (hunk 2 above)
Comment (29riyasaxena) [reply python/grass/jupyter/baseseriesmap.py:5]: Yes, I have modified it.

**Review by petrasovaa — APPROVED** [review]

**Pull request merged**